app/controllers/webauthn_verifications_controller.rb

```diff
@@ -49,8 +48,7 @@ def set_verification
     @verification = WebauthnVerification.find_by(path_token: webauthn_token_param)
 
     render_not_found and return unless @verification
-    # TODO: Return a message if format is plain text
-    redirect_to root_path, alert: t("webauthn_verifications.expired_or_already_used") if @verification.path_token_expired?
+    render_expired and return if @verification.path_token_expired?
```

Comment:
I dont think the `and return` does anything here, since it's the last expression of the method anyways